### PR TITLE
stolostron/backlog#20772

### PR DESCRIPTION
**Pull request by swopebe**
--wokring MCE plugin doc

### CODE REVIEW

File: multicluster_engine/console_mce.adoc [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[#mce-console-overview]
+= Console overview
+
+{ocp-short} console plug-ins are available with {ocp-short} 4.10 web console and can be integrated. To use this feature, the console plug-ins must remain enabled. The multicluster engine operator displays certain console features without {product-title-short} installed. If you install {product-title-short}, you see more console capability.
```
Comment (KevinFCormier) [inline multicluster_engine/console_mce.adoc]: We might want to mention that these features are the ones from ACM under the **Infrastructure** and **Credentials** navigation items.

File: multicluster_engine/console_mce.adoc [hunk 2]
```diff
@@ -0,0 +1,15 @@
+[#mce-console-overview]
+= Console overview
+
+{ocp-short} console plug-ins are available with {ocp-short} 4.10 web console and can be integrated. To use this feature, the console plug-ins must remain enabled. The multicluster engine operator displays certain console features without {product-title-short} installed. If you install {product-title-short}, you see more console capability. 
+
+
+. To disable the plug-in, be sure you are in the _Administrator_ perspective in the {ocp-short} console.
+. Find *Administration* in the navigation and click *Cluster Settings*, then click _Configuration_ tab. 
+. From the list of _Configuration resources_, click the **Console** resource with the `operator.openshift.io` API group, which contains cluster-wide configuration for the web console. 
+. Click on the _Console plug-ins_ tab. Both the `acm` and `mce` plug-ins are listed.
```
Comment (KevinFCormier) [inline multicluster_engine/console_mce.adoc]: `acm` is only listed if ACM is installed.

File: multicluster_engine/console_mce.adoc [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+
+. To disable the plug-in, be sure you are in the _Administrator_ perspective in the {ocp-short} console.
+. Find *Administration* in the navigation and click *Cluster Settings*, then click _Configuration_ tab. 
+. From the list of _Configuration resources_, click the **Console** resource with the `operator.openshift.io` API group, which contains cluster-wide configuration for the web console. 
+. Click on the _Console plug-ins_ tab. Both the `acm` and `mce` plug-ins are listed. 
+. Modify plug-in status from the table. In a few moments, you are prompted to refresh the console.
+
+For {ocp-short} 4.10 with the plug-ins enabled, you can access {product-title-short} within the {ocp-short} console from the cluster switcher by selecting *All Clusters* from the drop-down menu. Without plug-ins enabled on {ocp-short} versions 4.8 through 4.10, {product-title-short} is available in the perspective switcher. You can only view certain features
```
Comment (KevinFCormier) [inline multicluster_engine/console_mce.adoc]: This part seems not relevant to multicluster engine. ACM would be available in the perspective switcher only if ACM is installed.

Last sentence is missing a period and seems vague. I'm not sure which scenario it is referring to.

**Review by KevinFCormier — CHANGES_REQUESTED** [review]
Thanks for adding this! I think there are a few spots that need clarification.

File: multicluster_engine/console_mce.adoc (hunk 3 above)
Comment (swopebe) [reply multicluster_engine/console_mce.adoc]: I think the install team wanted this part in--I don't remember who asked for it. They want users on 4.8 to understand why the view is different. 

I moved it to a note.

We can remove it or we can let OCP decide what to do with it when we send it off next release.

File: multicluster_engine/console_mce.adoc (hunk 2 above)
Comment (swopebe) [reply multicluster_engine/console_mce.adoc]: The `mce` plug-in is listed. *Note:* If {product-title-short} is installed, it is also listed as `acm`.

File: multicluster_engine/console_mce.adoc (hunk 3 above)
Comment (KevinFCormier) [reply multicluster_engine/console_mce.adoc]: OK, that's better since you removed the last sentence. With ACM, the feature set is not limited.

**Review by KevinFCormier — APPROVED** [review]

**Pull request merged**